internal/pkg/performance/metrics.go

```diff
@@ -20,11 +21,13 @@ var once sync.Once
 const (
 	STDOUT = "stdout"
 	STDERR = "stderr"
+
+	notRuningState = 0
+	runningState   = 1
```

Comment:
There's stuff in this PR that doesn't seem to be related to vendorization?

Reply:
reverted this change